### DISCUSSION TITLE
Add instagram bro

### DISCUSSION THREAD

**Discussion by AshutoshGoswami24** [LISA-KOREA/UPLOADER-BOT-V4#16] [Q&A] [answered]
ERROR... ERROR: [Instagram] C6oqCVkMFJz: Requested content is not available, rate-limit reached or login required. Use --cookies, --cookies-from-browser, --username and --password, --netrc-cmd, or --netrc (instagram) to provide account credentials

**Comment by LISA-KOREA** [comment, marked as answer]
No i create new one only instagram reels and pics download bot

**Reply by AshutoshGoswami24** [reply]
ok  thenks for reply
